config/kloubak2-subt-estop-lora-jetson.json

```diff
@@ -213,6 +213,7 @@
               ["kloubak.pose2d", "localization.odom"],
               ["imu.orientation", "localization.orientation"],
               ["localization.pose3d", "app.pose3d"],
+              ["kloubak.pose2d", "app.pose2d"],
```

Comment:
I would keep it in separate PR, but  ... keep it if you want (but fix typo "necoders" plz